### PR TITLE
CEPH-9220[RGW]:Create users with different permission levels and acce…

### PR DESCRIPTION
**Pull request by ckulal**
…ss the versioned buckets

Automation: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-9220

[RGW]: Create users with different permission levels and access the versioned buckets
Pass log:
Read: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/user_permission/user_perm_read
Write: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/user_permission/user_perm_write
Redadwrite: http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/user_permission/user_perm_readwrite

Pass logs for existing config which used enhanced script file:
http://magna002.ceph.redhat.com/ceph-qe-logs/Chaithra/user_permission/swift-configs/

### CODE REVIEW

**Review by anrao19 — CHANGES_REQUESTED** [review]
update config fines in config section of file
: https://github.com/red-hat-storage/ceph-qe-scripts/blob/1b650597a6c944332c3643955795eb81d809ac45/rgw/v2/tests/s3_swift/test_swift_basic_ops.py#L18

**Review by anrao19 — APPROVED** [review]
Changes LGTM

File: rgw/v2/tests/s3_swift/test_swift_basic_ops.py [hunk 1]
```diff
@@ -585,7 +590,7 @@ def test_exec(config, ssh_con):
                 log.info(f"Delete container {container_name} with container owner")
                 rgw.delete_container(container_name)
 
-        if config.test_ops.get("enable_version_by_s3", False):
+        elif config.test_ops.get("enable_version_by_s3", False):
```
Comment (mkasturi18) [inline rgw/v2/tests/s3_swift/test_swift_basic_ops.py:593]: @ckulal , can we retain using if statement, as we might endup executing the 'else' section for every run.

**Review by mkasturi18 — CHANGES_REQUESTED** [review]

File: rgw/v2/tests/s3_swift/test_swift_basic_ops.py (hunk 1 above)
Comment (ckulal) [reply rgw/v2/tests/s3_swift/test_swift_basic_ops.py]: done

**Review by mkasturi18 — COMMENTED** [review]
Changes looks good

**Review by mkasturi18 — APPROVED** [review]

**Pull request merged**